packages/babel-config/src/plugins/babel-plugin-redwood-routes-auto-loader-rsc-client.ts

```diff
@@ -0,0 +1,120 @@
+import type { PluginObj, types } from '@babel/core'
+
+import {
+  ensurePosixPath,
+  importStatementPath,
+  processPagesDir,
+} from '@redwoodjs/project-config'
+
+import {
+  getPathRelativeToSrc,
+  withRelativeImports,
+} from './babel-plugin-redwood-routes-auto-loader'
+
+export function RedwoodRoutesAutoLoaderRscClientPlugin({
```

Comment:
```suggestion
export function redwoodRoutesAutoLoaderRscClientPlugin({
```
I'd prefer this to start with a lowercase letter